plugins/gatsby-source-jenkinsplugins/.npmignore

```diff
@@ -0,0 +1,33 @@
+# Logs
```

Comment:
is this needed as well?

Reply:
Nope, left behind files from the source I forked it from